docs/source/install.rst

```diff
@@ -1,97 +1,94 @@
+prerequisite for installing Jupyter notebook. Select one of the scenarios to
+install Jupyter:
 
-* :ref:`New user: New to Python and Jupyter <new-to-python-and-jupyter>`
-* :ref:`Experienced Python user: New to Jupyter <existing-python-new-jupyter>`
-* :ref:`Upgrading Jupyter <upgrading>`
+    - :ref:`Jupyter (new to Python) <new-to-python-and-jupyter>`
+    - :ref:`Jupyter (experience with Python) <existing-python-new-jupyter>`
+    - :ref:`Upgrading a Jupyter installation <upgrading>`
 
 .. note::
 
-     All commands shown below must be run in the Terminal (Mac/Linux) or Command
-     Prompt (Windows)
+     All installation commands should be run in the Terminal (for Mac and Linux)
+     or the Command Prompt (Windows).
 
 .. _new-to-python-and-jupyter:
 
-New Users: New to Python and Jupyter
-------------------------------------
+Jupyter (new to Python)
+-----------------------
 
-If you're new to Python, we highly recommend `installing Anaconda
-<https://www.continuum.io/downloads>`_. Anaconda includes and conveniently
+For new users, we **highly recommend** `installing Anaconda
+<https://www.continuum.io/downloads>`_. Anaconda conveniently
 installs Python, the Jupyter Notebook, and other commonly used packages for
 scientific computing and data science. Follow Anaconda's instructions for
-downloading the Python 3.5 version.
+downloading and installing the Python 3.5 version.
 
-After you have Anaconda installed, you should make sure that Jupyter is up to
-date. Run the following command in the Terminal (Mac/Linux) or Command
-Prompt (Windows)::
+After installing Anaconda, run the following command in the Terminal or the
+Command Prompt to install Jupyter::
 
     conda install jupyter
 
-Alternatively, you may also use the Anaconda Launcher application to update to
-the latest version.
-
-Now, you may consider :ref:`installing additional kernels <installing-kernels>`
-and :ref:`next steps <next-steps>` such as running the Jupyter Notebook.
+Alternatively, use the Anaconda Launcher application to update to the latest
+Jupyter version.
 
+See :ref:`installing additional kernels <installing-kernels>` to use programming
+languages other than Python and :ref:`next steps <next-steps>` for running the
+Jupyter Notebook.
 
 .. _existing-python-new-jupyter:
 
-Experienced Python user: New to Jupyter
----------------------------------------
+Jupyter (experience with Python)
+--------------------------------
 
 .. important::
 
-    **Jupyter installation requires Python 3.3 or greater, or Python 2.7.**
+    **Prerequisite: Jupyter installation requires Python 3.3 or greater, or Python 2.7.**
 
-    If you need to use Python 2.6 or 3.2, you may install
+    If Python 2.6 or 3.2 is needed, install
     `IPython 1.2 <http://archive.ipython.org/release/1.2.0/>`_.
-    Older releases of IPython are available `here <http://archive.ipython.org/release/>`__.
+    Older releases of IPython are available
+    `here <http://archive.ipython.org/release/>`__.
```

Comment:
Given how ancient 1.2 is, maybe we should cut this paragraph entirely. Python 3.3 and 2.7 have been out for long enough to be pretty ubiquitous by now.